src/main/resources/META-INF/squbs-meta.conf

```diff
@@ -13,3 +13,8 @@ squbs-actors = [
     name = sample
   }
 ]
+squbs-extensions = [
```

Comment:
Hmmm, this should not be in user code. We need to take care the shutdown is proper. Is there any use case user code won't want this?